aiida_mlip/calculations/base.py

```diff
@@ -211,8 +211,8 @@ def prepare_for_submission(
         # Transform the structure data in xyz file called input_filename
         input_filename = self.inputs.metadata.options.input_filename
         atoms = structure.get_ase()
-        # with folder.open(input_filename, mode="w", encoding='utf8') as file:
-        write(folder.abspath + "/" + input_filename, images=atoms)
+        with folder.open(input_filename, mode="w", encoding=None) as file:
```

Comment:
Why `None`?

Reply:
Probably can put it back to utf8, I was trying things and thought that could be a problem but prob not